library/src/main/java/com/telefonica/mistica/compose/card/postercard/PosterCard.kt

```diff
@@ -0,0 +1,94 @@
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.telefonica.mistica.compose.tag.Tag
+import com.telefonica.mistica.compose.theme.MisticaTheme
+
+@Composable
+fun PosterCard(
+    modifier: Modifier = Modifier,
+    aspectRatio: PosterCardAspectRatio,
+    backgroundType: PosterCardBackgroundType,
+    headline: Tag? = null,
+    preTitle: String? = null,
+    title: String? = null,
+    subtitle: String? = null,
+    description: String? = null,
+    firstTopAction: TopActionData? = null,
+    secondTopAction: TopActionData? = null,
+    onClickAction: (() -> Unit)? = null,
+    customContent: (@Composable () -> Unit)? = null,
+) {
+    BoxWithConstraints(modifier = modifier) {
+        androidx.compose.material.Card(
+            elevation = 0.dp,
+            backgroundColor = Color.Red,
+            shape = RoundedCornerShape(16.dp),
+            border = BorderStroke(width = 1.dp, color = MisticaTheme.colors.border),
+            modifier = Modifier
+                .width(maxWidth)
+                .let { modifierValue ->
+                    onClickAction?.let {
+                        modifierValue.clickable { it.invoke() }
+                    }
+                    modifierValue.heightIn(
+                        min = maxWidth / aspectRatio.ratio,
+                        max = Dp.Infinity
+                    )
+                }
+        ) {
+            PosterCardBackground(backgroundType = backgroundType) {
+                Column(
+                    modifier = Modifier.align(alignment = Alignment.BottomCenter),
+                    verticalArrangement = Arrangement.Bottom
+                ) {
+                    PosterCardMainContent(
+                        backgroundType = backgroundType,
+                        tag = headline,
+                        preTitle = preTitle,
+                        title = title,
+                        description = description,
+                        subtitle = subtitle,
+                        customContent = customContent
+                    )
+                }
+                if (firstTopAction != null || secondTopAction != null) {
+                    PosterCardTopActions(
+                        modifier = Modifier.align(alignment = Alignment.TopCenter),
+                        firstTopAction = firstTopAction,
+                        secondTopAction = secondTopAction
+                    )
+                }
+            }
+        }
+    }
+
+}
+
+enum class PosterCardAspectRatio(val ratio: Float) {
+    AR_AUTO(ratio = Float.NaN),
+    AR_1_1(ratio = 1f),
+    AR_7_10(ratio = 0.7f),
+    AR_9_10(ratio = 0.9f),
+    AR_16_9(16 / 9f)
+}
+
+sealed class PosterCardBackgroundType(open var inverseDisplay: Boolean) {
+    data class Image(val imageResource: Int, val contentDescription: String = "") : PosterCardBackgroundType(inverseDisplay = true)
```

Comment:
```suggestion
    data class Image(val imageResource: Int, val contentDescription: String? = null) : PosterCardBackgroundType(inverseDisplay = true)
```
Same definition as you have done here: https://github.com/Telefonica/mistica-android/pull/398/files#diff-dd2b9dff03ae4ec8918409185caf888d1146bb8ba1b837173e163a58d6f319b7R76 :)